src/cli.js

```diff
@@ -68,21 +70,32 @@ function copyModelViewer(){
   const height = argv.height || 1024;
   const format = argv.image_format || 'image/png';
   const quality = argv.image_quality || 0.92;
+  const timeout = argv.timeout || 10000;
 
   const {page, browser} = await startBrowser({width, height, libPort: libServer.port});
 
   const t2 = performance.now();
   INFO("--- Started puppeteer browser", `(${timeDelta(t1, t2)} s)`);
 
+  page.exposeFunction('logInfo', (message) => {
+    INFO(message)
+  });
+  
+  let t3 = performance.now();
   await loadGLBAndScreenshot(page, {
     glbPath,
     outputPath: argv.output,
     format,
     quality,
+    timeout,
+  }).then(()=>{
+    t3 = performance.now();
+    INFO("--- Took snapshot of", argv.input, `(${timeDelta(t2, t3)} s)`);
+  }).catch(()=>{
+    t3 = performance.now();
+    INFO("--- Failed to take snapshot of", argv.input, `(${timeDelta(t2, t3)} s)`);
```

Comment:
The program previously would just crash on failure this add some error messaging.